index.js

```diff
@@ -7,7 +7,7 @@ var hash = function (str) {
 
     while (i) hash = (hash * 33) ^ str.charCodeAt(--i);
 
-    return '_' + (hash >>> 0).toString(36);
+    return (hash >>> 0).toString(36);
```

Comment:
I remember there was some kinda reason for `_`. Not sure now what it was, maybe it was something that class names cannot start with a digit and, if prefix is empty string, generated hashes would not be valid class names (if I remember correctly).

If that is the case, maybe we can always enforce a non-empty, non-leading-digit prefix that we check in development?

Reply:
That’s what I was thinking, yeah. The default prefix is `_` which already solves for this. You could warn if somebody specifies and invalid prefix.